docs/_layouts/default.html

```diff
@@ -55,6 +55,17 @@
               <li><a href="{{ '/debugging.html' | relative_url }}">Debugging common Bolt errors</a></li>
             </ul>
           </li>
+          <li>
+            Bolt Developer Updates
+            <ul>
+              {% for post in site.posts %}
+                <li><a href="{{ post.url | remove: "/index.html" | relative_url }}">{{ post.date | date_to_string: "ordinal", "US" }} - {{ post.title }}</a></li>
+              {% endfor %}
+              {% for post in site.drafts %}
```

Comment:
Do we want to have drafts visible?

Reply:
Whoops, good catch. This was me trying to debug why one of my posts wasn't showing up (turns out it had a date in the future).